alicja/config/proto/interface.proto

```diff
@@ -9,13 +9,13 @@ option go_package = "github.com/googleinterns/step224-2020/config/proto"
 import "github.com/googleinterns/step224-2020/config/proto/targets.proto";
```

Comment:
- [ ] Swap the order of line 9 and 7.  Import statements should come before options.
- [ ] Add a ; at the end of every option statement